src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -39,8 +39,11 @@
 Some part of this code refers to
 https://github.com/microsoft/vscode-extension-samples/blob/2556c82cb333cf65d372bd01ac30c35ea1898a0e/custom-editor-sample/src/catScratchEditor.ts
 */
+
+import * as fs from 'fs';
```

Comment:
can we use `vscode.workspace.fs`?
ref: https://github.com/Samsung/ONE-vscode/issues/746